scm/github.go

```diff
@@ -595,14 +581,14 @@ func (s *GithubSCM) createTeam(ctx context.Context, opt *TeamOptions) (*Team, er
 			Name: opt.TeamName,
 		})
 		if err != nil {
-			if opt.TeamName != TeachersTeam && opt.TeamName != StudentsTeam {
+			if opt.TeamName != TeachersTeam {
 				return nil, ErrFailedSCM{
 					Method:   "CreateTeam",
 					Message:  fmt.Sprintf("failed to create GitHub team %s, make sure it does not already exist", opt.TeamName),
 					GitError: fmt.Errorf("failed to create GitHub team %s: %w", opt.TeamName, err),
 				}
 			}
-			// continue if it is one of standard teacher/student teams. Such teams can be safely reused
+			// continue if it is the standard teacher team that can be safely reused
```

Comment:
Why is it being reused?

Reply:
I think the original idea was as follows: we don't want the method to fail if there already is a teacher/student team in the organization and can just use the existing one. But I'm not sure this is what this method currently does.